frontend/pages/admin/IntegrationsPage/cards/MdmSettings/VppPage/components/RenewVppModal/RenewVppModal.tsx

```diff
@@ -82,14 +79,7 @@ const RenewVppModal = ({
           graphicName="file-vpp"
           buttonType="link"
           buttonMessage="Upload"
-          filePreview={
-            tokenFile && (
-              <FileDetails
-                details={{ name: tokenFile.name }}
-                graphicName="file-vpp"
-              />
-            )
-          }
+          fileDetails={tokenFile ? { name: tokenFile.name } : undefined}
```

Comment:
Looks like the graphic name may have been accidentally removed here?

Reply:
Passed in from line 79